src/main/java/control/DetailViewController.java

```diff
@@ -40,13 +39,22 @@
     private DetailView detailView;
     private ControllerManager manager;
 
+    @Getter
```

Comment:
I believe in the rest of the code, we are using @Getter and @Setter on the same line. Could you conform to this standard?